packages/app/src/systems/Chains/eth/machines/assetsMachine.tsx

```diff
@@ -0,0 +1,180 @@
+import type { StateFrom } from 'xstate';
+import { assign, createMachine } from 'xstate';
+
+import { AssetService } from '../services';
+import { ethLogoSrc } from '../utils';
+
+import type { BridgeAsset } from '~/systems/Bridge';
+import { FetchMachine } from '~/systems/Core';
+
+export const AssetList = [
+  {
+    address:
+      '0x0000000000000000000000000000000000000000000000000000000000000000',
+    symbol: 'ETH',
+    image: ethLogoSrc,
+    decimals: 18,
+  },
+];
```

Comment:
I think would be good to:
create a `constants.ts` file in Chains/eth

then export 
```
export const ETH_NATIVE_ASSET = {
    address:
      '0x0000000000000000000000000000000000000000000000000000000000000000',
    symbol: 'ETH',
    image: ethLogoSrc,
    decimals: 18,
  }
```